python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -144,7 +145,12 @@ def measure(self, measRecord, exposure):
         xmy2 = xmy*xmy
         xy2 = Ixy*Ixy
         a2 = 0.5 * (xpy + np.sqrt(xmy2 + 4.0*xy2))
-        sigma = exposure.getPsf().getSigma()
+
+        # Get the width of the PSF at the center of the trail
+        center = Point2D(xc, yc)
```

Comment:
Somewhat outside of this ticket review, but is there a reason to get `xc` and `yc` from the hard-coded `base_SdssShape` plugin, instead of getting `center = measRecord.getCentroid()`?